docs/package-sets.md

```diff
@@ -1,12 +1,18 @@
 #### Nixpkgs CUDA and ROCm
 
-Built on `nixos-unstable-small` for `x86_64-linux`, `aarch64-linux`.
-
 [CUDA and ROCm release set in nixpkgs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/release-cuda.nix)
 
-[https://hydra.nix-community.org/jobset/nixpkgs/cuda](https://hydra.nix-community.org/jobset/nixpkgs/cuda)
+Built on `nixos-unstable-small` for `x86_64-linux`:
+
+- [https://hydra.nix-community.org/jobset/nixpkgs/cuda](https://hydra.nix-community.org/jobset/nixpkgs/cuda)
+
+Built on `nixos-unstable-small` for `x86_64-linux`:
+
+- [https://hydra.nix-community.org/jobset/nixpkgs/rocm](https://hydra.nix-community.org/jobset/nixpkgs/rocm)
```

Comment:
Is rocm + aarch64-linux something that people would actually be using?

Reply:
I'll just drop all of the aarc64 builds for now, we can add them back if they are useful.